suites/squid/nvmeof/tier-3_8-1-nvmeof_4-group_8-gw_scale.yaml

```diff
@@ -0,0 +1,519 @@
+# Scale test suite including all new features from 8.1
+# GW group 1 with 8 GWs and 76 subsystems with 1520 namespaces - all new features of 8.1
+# GW group 2 with 8 GWs and 76 subsystems with 1520 namespaces - all new features of 8.1 + set_qos
+# GW group 3 with 8 GWs and 76 subsystems with 1520 namespaces - No Inband , vmware use cases
+# GW group 4 with 8 GWs and 76 subsystems with 1520 namespaces - scale with no huge pages
+# 5 initiator nodes
+# Bare Metal conf TBD
+
+tests:
+  - test:
+      abort-on-fail: true
+      module: install_prereq.py
+      name: install ceph pre-requisites
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                mon-ip: node1
+                registry-url: registry.redhat.io
+                allow-fqdn-hostname: true
+                log-to-file: true
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      desc: RHCS cluster deployment using cephadm
+      destroy-clster: false
+      module: test_cephadm.py
+      name: deploy cluster
+
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        nodes:
+          - node41
+          - node42
+          - node43
+          - node44
+          - node45
+        install_packages:
+          - ceph-common
+        copy_admin_keyring: true
+      desc: Setup client on NVMEoF gateway
+      destroy-cluster: false
+      module: test_client.py
+      name: configure Ceph client for NVMe tests
+      polarion-id: CEPH-83573758
+  - test:
```

Comment:
suggesting to call this module as and when required for the pool and service creation. or in any specific test case.
we can combine `deploy NVMeoF service for 4 GW groups with 8 GWs` and `create rbd pools for nvmeof service and data` in one test case and try to call it from any one of the test case.